cmd/kyverno/main.go

```diff
@@ -483,10 +488,20 @@ func main() {
 		os.Exit(1)
 	}
 
+	pInformer.Start(stopCh)
+	kubeInformer.Start(stopCh)
+	kubeKyvernoInformer.Start(stopCh)
+	kubedynamicInformer.Start(stopCh)
+	pInformer.WaitForCacheSync(stopCh)
+	kubeInformer.WaitForCacheSync(stopCh)
+	kubeKyvernoInformer.WaitForCacheSync(stopCh)
+	kubedynamicInformer.WaitForCacheSync(stopCh)
```

Comment:
Dynamic informers?

Reply:
Looks like we can remove them entirely.
Done.